modules/miner/blockmanager.go

```diff
@@ -158,6 +162,14 @@ func (m *Miner) managedSubmitBlock(b types.Block) error {
 
 // SubmitHeader accepts a block header.
 func (m *Miner) SubmitHeader(bh types.BlockHeader) error {
+	if err := m.tg.Add(); err != nil {
```

Comment:
The typical pattern is to check the return value of `tg.Add`. If you used `IsStopped` first, you'd still need to check the return value of `tg.Add` anyway. See https://github.com/NebulousLabs/Sia/blob/master/modules/gateway/peers.go#L142